docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -275,156 +275,218 @@ RAJA policies for GPU execution using CUDA or HIP are essentially identical.
+                                           launch (loop) directly to GPU threads
+                                                         in x-dimension, one
+                                                         iterate per thread
+                                                         (see note below about
+                                                         limitations)
+ cuda/hip_thread_y_direct                  kernel (For)  Same as above, but map
+                                           launch (loop) to threads in y-dim
+ cuda/hip_thread_z_direct                  kernel (For)  Same as above, but map
+                                           launch (loop) to threads in z-dim
+ cuda/hip_thread_x_loop                    kernel (For)  Similar to
+                                           launch (loop) thread-x-direct
+                                                         policy, but use a
+                                                         block-stride loop which
+                                                         doesn't limit number of
+                                                         loop iterates, this is
+                                                         not safe to use with
+                                                         Cuda/HipSyncThreads
+ cuda/hip_thread_y_loop                    kernel (For)  Same as above, but for
+                                           launch (loop) threads in y-dimension
+ cuda/hip_thread_z_loop                    kernel (For)  Same as above, but for
+                                           launch (loop) threads in z-dimension
+ cuda/hip_thread_syncable_loop<dims...>    kernel (For)  Similar to thread-loop
+                                           launch (loop) policy, but safe to use
+                                                         with Cuda/HipSyncThreads
+ cuda/hip_thread_size_x_direct<size>       kernel (For)  Same as thread_x_direct
+                                           launch (loop) policy above but with
+                                                         a compile time number of
+                                                         threads
+ cuda/hip_thread_size_y_direct<size>       kernel (For)  Same as above, but map
+                                           launch (loop) to threads in y-dim
+ cuda/hip_thread_size_z_direct<size>       kernel (For)  Same as above, but map
+                                           launch (loop) to threads in z-dim
+ cuda/hip_flatten_threads_{xyz}_direct     launch (loop) Reshapes threads in a
+                                                         multi-dimensional thread
+                                                         team into one-dimension,
+                                                         accepts any permutation
+                                                         of dimensions
+                                                         (expt namespace)
```

Comment:
Is this still in the 'expt' namespace?

Reply:
It is not in expt anymore.

Reply:
That's what I thought. But I was in a meeting and didn't check. Please remove that from the docs. Thank you.

Reply:
removed